lisa/operating_system.py

```diff
@@ -1417,7 +1417,7 @@ def _initialize_package_installation(self) -> None:
             # refer https://www.centos.org/centos-linux-eol/
             # CentOS 8 is EOL, old repo mirror was moved to vault.centos.org
             # CentOS-AppStream.repo, CentOS-Base.repo may contain non-existed repo
-            # use skip_if_unavailable to aviod installation issues bring in by above
+            # use skip_if_unavailable to avoid installation issues bring in by above
```

Comment:
[minor] - If we are here correcting the spelling then might as well change this - 
"issue brought in by above"